configs/default/MTKS-MATEKF405TE.config

```diff
@@ -0,0 +1,149 @@
+# Betaflight / STM32F405 (S405) 4.3.0 Jan 16 2022 / 12:27:21 (ee740d1) MSP API: 1.44
+
+board_name MATEKF405TE
+manufacturer_id MTKS
+
+# resources
+resource BEEPER 1 B09
+resource MOTOR 1 C09
+resource MOTOR 2 C08
+resource MOTOR 3 B15
+resource MOTOR 4 A08
+resource MOTOR 5 B11
+resource MOTOR 6 B10
+resource MOTOR 7 B03
+resource MOTOR 8 A15
+resource SERVO 1 B14
+resource SERVO 2 A06
+resource SERVO 3 B06
+resource PPM 1 A03
+resource LED_STRIP 1 B01
+resource SERIAL_TX 1 A09
+resource SERIAL_TX 2 A02
+resource SERIAL_TX 3 C10
+resource SERIAL_TX 4 A00
+resource SERIAL_TX 5 C12
+resource SERIAL_TX 6 C06
+resource SERIAL_TX 11 A02
```

Comment:
need to delete this command, otherwise SERIAL_TX 2 A02 will be overridden after loading config. they can be coexisted in legacy target, but not in unified target.

Reply:
Removed line 27